### PR TITLE
Add a split type for classifier FTC visibility

### PR DESCRIPTION
**Pull request by zwolf**
Adds a new split type for first-to-classify events in the classifier. This is similar to the existing subject viewer message split, but will allow toggling of banners and the like.

Not sure who to tag on this for review, so I'll throw it to the wind.

### CODE REVIEW

File: app/models/concerns/metric_types.rb [hunk 1]
```diff
@@ -18,6 +18,10 @@ module MetricTypes
     ],
     'subject.first-to-classify' => [
       'classification_created'
+     ],
+    'classifier.first-to-classify.visible' => [
```
Comment (camallen) [inline app/models/concerns/metric_types.rb]: OOI is this the first person to ever classify on the project / workflow OR the first classification from each user. If the latter maybe reword the name, `user.first-classification-event` or something.

File: spec/models/split_spec.rb [hunk 2]
```diff
@@ -182,10 +182,15 @@ def expect_assignment(suv)
         it{ is_expected.to match_array mini_course_visible_metrics }
       end
 
-      context 'for first to classify subject' do
+      context 'for first to classify subject message' do
         subject{ subject_first_to_classify.metric_types }
         it{ is_expected.to match_array subject_first_to_classify_metrics }
       end
+
+      context 'for first to classify classifier visiblity' do
```
Comment (camallen) [inline spec/models/split_spec.rb]: this naming could be better, i'm not sure what this is testing

File: app/models/concerns/metric_types.rb (hunk 1 above)
Comment (srallen) [inline app/models/concerns/metric_types.rb]: It's an a/b split for a banner that notifies the user that they are first to classify the current subject. I tried to follow the convention Parrish seemed to be going with documented here: https://github.com/zooniverse/Seven-Ten/blob/master/docs/variants.md#attributes

The banner is in the classifier, the split is for first to classify banner, and it's a visibility split. I'm open to a more clear naming if someone can come up with one that seems to follow that convention already in place.

Comment (srallen) [inline app/models/concerns/metric_types.rb]: Looking at that variant key list again, I suppose it could be `subject.first-to-classify.visible`. `subject.first-to-classify` is a text split that's in the classification summary.

**Comment by srallen** [comment]
@zwolf thinking it over, let's change the experiment name to `subject.first-to-classify.visible` and then I think this will be good to go.
